recipe/recipe.yaml

```
@@ -0,0 +1,51 @@
+schema_version: 1
+
+context:
+  name: vector
+  version: 1.6.1
+  python_min: 3.8
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/vector-${{ version }}.tar.gz
+  sha256: 59982d0ce553d35f76c63396b57d7fcf5004deab33270bfdb6e54c65db62d6e4
+
+build:
+  number: 1
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python 3.8.*
```

```suggestion
    - python ${{ python_min }}.*
```

Currently there's a bug for all of these that the `python_min` doesn't get properly replaced everywhere, so you need to manually replace the `${{ python_min}}.*` in places like this.

c.f. https://github.com/conda-incubator/conda-recipe-manager/issues/307